src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/claims/ClaimsControllerTest.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.reform.civil.controllers.claims;
+
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpHeaders;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.civil.controllers.BaseIntegrationTest;
+import uk.gov.hmcts.reform.civil.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.civil.service.CoreCaseDataService;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class ClaimsControllerTest extends BaseIntegrationTest {
+
+    private static final String CLAIMS_URL = "/claims/{claimId}";
+
+    @MockBean
+    private CoreCaseDataService coreCaseDataService;
+
+    @MockBean
+    private CaseDetailsConverter caseDetailsConverter;
+
+    @Test
+    @SneakyThrows
+    public void shouldReturnHttp200() {
+        CaseDetails expectedCaseDetails = CaseDetails.builder().id(1L).build();
+        when(coreCaseDataService.getCase(1L))
+            .thenReturn(expectedCaseDetails);
+
+        mockMvc.perform(
+            get(CLAIMS_URL, 1L)
+                .header(HttpHeaders.AUTHORIZATION, BEARER_TOKEN))
+                .andDo(print())
+                .andExpect(status().isOk());
```

Comment:
Is it worth adding an assertion on the id of the returned object? (should be equal to 1)